manimpango/cmanimpango.pyx

```diff
@@ -138,22 +140,28 @@ class MarkupUtils:
         slant: str,
         weight: str,
         size: int,
-        line_spacing: int,
+        _: int, # for some there was a keyword here.
```

Comment:
@PhilippImhof though on what can be done here? It looks really weird.

Reply:
Or we remove all these crazy things and tag the next major release so that it doesn't break things.

Reply:
I have to admit that I do not quite see the problem behind this change...?

Reply:
well, the problem here is accepting an extra positional argument (which does nothing the `_` one), for compatibility with Manim.
Before this, there used to be a line_spacing argument which isn't used anywhere else, which I moved to keyword one argument below this.
Also, the previously passed `line_spacing` by Manim, seems to produce weird results.
Thoughts on removing this compatibility code and tagging `v0.3.0`?

Reply:
It's true that `line_spacing` is not used in `MarkupText`'s variant of `text2svg`. Replacing it by `_` for backwards compatibility seems sensible. I think we do not even have to keep it as a keyword argument.

Reply:
> I think we do not even have to keep it as a keyword argument.

This PR adds an option to pass `line_spacing` to Pango so that it works :).